app/interfaces/api/entities/search_result.rb

```diff
@@ -37,6 +37,7 @@ class SearchResult < BaseEntity
         expose :injection_errored
         expose :cav_warning
         expose :supplementary
+        expose :lgfs_hardship
```

Comment:
@kmahern once the agfs branch is in master with the agfs filter will it not be possible to simply use the `:hardship` filter you already added to that branch

Reply:
I think this might be OK? I have  added expose :hardship in the agfs branch, so I don't think there should be a conflict (although possibly I should change this to expose :agfs_hardship)?

Reply:
if it can't be shared then yes the agfs equivalent should be named agfs_hardship (we do this for warrant filters by the looks of it). you could PR against my branch

Reply:
👍 see https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3282